docs/cudf/source/user_guide/performance_comparisons.ipynb

```diff
@@ -0,0 +1,1568 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    gdf
</pre><p>Combine with the above cell. No need to show both pdf and gdf outputs. Readers will trust that's correct.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/cudf/pull/12595/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>